Makefile

```
@@ -291,7 +291,10 @@ dev: IMAGE_REPO=localhost:$(REGISTRY_PORT)/ec
 dev: PODMAN_OPTS=--tls-verify=false
 dev: TASK_REPO=localhost:$(REGISTRY_PORT)/ec-task-bundle
 dev: SKOPEO_ARGS=--src-tls-verify=false --dest-tls-verify=false
-dev: TASKS:=$(shell T=$$(mktemp) && yq e ".spec.steps[].image? = \"localhost:$(REGISTRY_PORT)/ec\"" tasks/verify-enterprise-contract/*/verify-enterprise-contract.yaml | yq 'select(. != null)' > "$${T}" && echo "$${T}")
+dev: TASKS:=$(shell T=$$(mktemp) && yq e ".spec.steps[].image? = \"localhost:$(REGISTRY_PORT)/ec\"" \
+    tasks/verify-enterprise-contract/*/verify-enterprise-contract.yaml \
+    tasks/verify-conforma-konflux-ta/0.1/verify-conforma-konflux-ta.yaml \
```

To be consistent, how about this:

```suggestion
    tasks/verify-conforma-konflux-ta/*/verify-conforma-konflux-ta.yaml \
```

---

..or change line 295 to specify the path explicitly.